Tests/Stream/Common/TestPayload.cs

```diff
@@ -19,19 +19,19 @@ public enum TaskType
       PayloadCheckSum,
     }
 
-    public byte [] DataBytes { get; set; }
+    public byte []? DataBytes { get; set; }
 
     public TaskType Type { get; set; }
 
-    public string ResultKey { get; set; }
+    public string? ResultKey { get; set; }
 
     public byte[] Serialize()
     {
       var jsonString = JsonSerializer.Serialize(this);
       return Encoding.ASCII.GetBytes(StringToBase64(jsonString));
     }
 
-    public static TestPayload? Deserialize(byte[] payload)
+    public static TestPayload? Deserialize(byte[]? payload)
```

Comment:
This nullable type declaration is still relevant after our changes?

Reply:
It should. It does not consider the same kind of payload. This is a object that is serialised and used as payload. It may be interesting to check that this ? is still relevant after our changes.